src/Categories/Category/Construction/Monoids.agda

```diff
@@ -0,0 +1,74 @@
+{-# OPTIONS --without-K --safe #-}
+open import Categories.Category.Core
+open import Categories.Category.Monoidal.Core
+
+module Categories.Category.Construction.Monoids {o ℓ e} {𝒞 : Category o ℓ e} (C : Monoidal 𝒞) where
```

Comment:
Perhaps this should be called ...`Construction.InternalMonoids` ?  The point is that the category of monoids (in `Setoid`, as they exist in stdlib) would also be expected to have that name. The file can still define `Monoids` as you have it, i.e. without changing the name of the category.

Reply:
Hmm, I'm not entirely convinced. I'd expect that to be in `Categories.Category.Instance.Monoids`. Perhaps I could add a note of explanation, and if/when that gets added, the two modules can refer to each other?

Reply:
Good point!  `Construction` vs `Instance` is indeed an appropriate differentiator.  A note would be splendid.